### PR TITLE
CRAFTERCMS-3231;CRAFTERCMS-3236

### PR DESCRIPTION
**Pull request by dejan-brkic**
Updated upgrade manager pipeline

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/3231
https://github.com/craftercms/craftercms/issues/3236

### CODE REVIEW

File: src/main/resources/crafter/studio/upgrade/3.1.x/3.1.0.27/site/site-config-tools-6.xslt [hunk 1]
```diff
@@ -38,7 +38,7 @@
         <xsl:copy>
             <xsl:apply-templates/>
             <xsl:if test="not(control/name = 'time')">
-                <xsl:element name="datasource">
```
Comment (jdrossl) [inline src/main/resources/crafter/studio/upgrade/3.1.x/3.1.0.27/site/site-config-tools-6.xslt:41]: Why did this change? was it a bug?

File: src/main/resources/crafter/studio/upgrade/pipelines.yaml [hunk 2]
```diff
@@ -412,6 +412,14 @@ pipelines:
           template: crafter/studio/upgrade/3.1.x/3.1.0.33/system/sample-notification-config.xslt
           commitDetails: Add pull from remote conflict notification
         - type: dbVersionUpgrader
+    - currentVersion: 3.1.0.33
+      nextVersion: 3.1.0.34
+      operations:
+        - type: xsltFileUpgrader
+          path: /configuration/samples/sample-contextual-nav.xml
```
Comment (jdrossl) [inline src/main/resources/crafter/studio/upgrade/pipelines.yaml]: Is this path right? shouldn't it be `site-config-tools.xml`?

File: src/main/resources/crafter/studio/upgrade/3.1.x/3.1.0.27/site/site-config-tools-6.xslt (hunk 1 above)
Comment (dejan-brkic) [reply src/main/resources/crafter/studio/upgrade/3.1.x/3.1.0.27/site/site-config-tools-6.xslt:41]: bug nobody noticed

File: src/main/resources/crafter/studio/upgrade/pipelines.yaml [hunk 3]
```diff
@@ -412,6 +412,14 @@ pipelines:
           template: crafter/studio/upgrade/3.1.x/3.1.0.33/system/sample-notification-config.xslt
           commitDetails: Add pull from remote conflict notification
         - type: dbVersionUpgrader
+    - currentVersion: 3.1.0.33
+      nextVersion: 3.1.0.34
+      operations:
+        - type: xsltFileUpgrader
+          path: /configuration/samples/sample-site-config-tools.xml
+          template: crafter/studio/upgrade/3.1.x/3.1.0.34/system/sample-site-config-tools.xslt
+          commitDetails: Add quick create menu item
```
Comment (jdrossl) [inline src/main/resources/crafter/studio/upgrade/pipelines.yaml]: Please update the comments according to the new changes

**Review by jdrossl — APPROVED** [review]

**Pull request merged**